src/mpyl/steps/deploy/k8s/resources/dagster.py

```diff
@@ -42,16 +56,18 @@ def to_user_code_values(
     for sealed_secret_env in builder.get_sealed_secret_as_env_vars():
         sealed_secret_env.value_from.secret_key_ref.name = release_name
         sealed_secret_refs.append(to_dict(sealed_secret_env, skip_none=True))
-
-    sealed_secret_manifest = builder.to_sealed_secrets()
-    sealed_secret_manifest.metadata.name = release_name
-
-    extra_manifests = (
-        {"extraManifests": [to_dict(sealed_secret_manifest)]}
-        if len(sealed_secret_refs) > 0
-        else {}
+    extra_manifests = collect_extra_manifests(
+        builder, len(sealed_secret_refs), release_name
     )
 
+    image_dict: dict = {
+        "pullPolicy": "Always",
+        "tag": run_properties.versioning.identifier,
+        "repository": f"{docker_registry.host_name}/{project.name}",
+    }
+    if docker_registry.host_name == "azure":
+        image_dict["imagePullSecrets"] = [{"name": "bigdataregistry"}]
```

Comment:
Should this be hardcoded like this instead of configurable like in the rest of mpyl from the mpyl_config in `docker.registries`?